feature-group-definitions/dialog.yml

```
@@ -1,4 +1,5 @@
 name: "<dialog>"
+description: The `<dialog>` HTML element represents a modal or non-modal dialog box.
```

How about an example or two?

```suggestion
description: The `<dialog>` HTML element represents a modal or non-modal dialog box, such as a confirmation prompt or inspector widget.
```

---

Not sure what an "inspector widget" is. Maybe:

```suggestion
description: The `<dialog>` HTML element represents a modal or non-modal dialog box, such as a confirmation prompt or a subwindow used to enter data.
```

---

Not a Mac user, eh? 😄 But yes, this works!